honeybee_schema/model.py

```diff
@@ -366,6 +374,40 @@ class Model(NamedBaseModel):
         description='The clockwise north direction in degrees.'
     )
 
+    units_system: UnitsSystems = Field(
+        default=UnitsSystems.meters,
+        description='Text for the units system in which the model geometry exists. '
+            'This is used to scale the geometry to the correct units for simulation '
+            'engines like EnergyPlus, which requires all geometry be in meters.'
+    )
+
+    tolerance: float = Field(
+        default=-1,
+        description='The maximum difference between x, y, and z values at which '
+            'vertices are considered equivalent. This value should be in the Model '
+            'units_system and is used in a variety of checks, including checks for '
+            'whether Room faces form a closed volume and subsequently correcting all '
+            'face normals point outward from the Room. Any negative values will result '
+            'in no attempt to evaluate whether the Room volumes is closed or check '
+            'face direction. So it is recommended that this always be a positive '
```

Comment:
What about 0 itself?

Reply:
I think the issue with that is that 0 is a real tolerance in some cases. It's an incredibly strict one that only few cases can meet without running into Python floating point issues. But I'm not sure we want to stop people from using 0 tolerance if they want. I guess Rhino doesn't allow 0 tolerance. And it is nice to have just one value that bypasses the tolerance check and not the whole set of negative numbers. I guess my only concern is that someone sees 0 and mistakes it for a really strict tolerance check rather than no tolerance check at all. Do you think there's any possibility of this happening?

Reply:
I guess if we're clear about it in the description, using 0 will be fine. Just to be sure that I don't sow confusion, though, I think I may go back and change the places that I actually used 0 as a real tolerance in the core libraries. Some of the [methods like this one](https://github.com/ladybug-tools/honeybee-core/blob/master/honeybee/face.py#L395) should be changed so that the tolerance is required rather than having an optional 0. Or maybe we just use a different default that's no 0 like what I'm doing here:
https://github.com/ladybug-tools/honeybee-core/blob/master/honeybee/face.py#L964

Reply:
I decided to go with 0 and enforce the following rules in honeybee-core:
1. 0 tolerance indicates cases where tolerance checks should be bypassed. I'm not considering it a "real" tolerance any more.
2. All honeybee-core methods requiring a tolerance input have a default value that is typical of building geometry in meters (0.01).
3. All ladybug-geometry methods requiring a tolerance will enforce that the user input the tolerance (so there's no default). I still have to implement this one completely but I've opened an issue for it: https://github.com/ladybug-tools/ladybug-geometry/issues/89